DSP Accel Library/slblocks.m

```diff
@@ -0,0 +1,13 @@
+function blkStruct = slblocks
```

Comment:
It would be great to make our library show up when typing `hdllib` at the command line. I'm not sure if that's possible, though. [hdllib](https://www.mathworks.com/help/hdlcoder/ref/hdllib.html) makes it so only hdl-compatible blocks are shown in the library browser.

Reply:
I'll look into this. Everything these blocks are made out of were from the hdllib compatible blocks, they should be able to show up.
I'll see how much it takes to make this happen.

Reply:
I tried looking into it briefly, but I couldn't find much. It seems like we may not be able to make this happen; there isn't much or any documentation on it, anyway. Maybe we'd have to write a modified version of `hdllib`.